### PR TITLE
input values: serialize and deserialize via JSDP

### PR DESCRIPTION
**Pull request by go-oleg**
Hook up juttled to use JSDP to serialize and deserialize input values it sends and receives.

JSDP supports JavaScript Date and moment.duration objects instead of juttle's JuttleMoment and JuttleMoment.duration so theres some translation logic in `jsdp-value-converter` to handle that.

Depends on https://github.com/juttle/juttle/pull/186 and https://github.com/juttle/juttle-jsdp/pull/2.

@mnibecker @mstemm

### CODE REVIEW

File: lib/jsdp-value-converter.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+var JuttleMoment = require("juttle/lib/moment").JuttleMoment;
+var moment = require("moment");
+var _ = require("underscore");
+
+var dateConverter = {
+    isConvertibleToJSDP: function(value) {
+        return value instanceof JuttleMoment && value.moment;
+    },
+    isConvertibleToJuttle: function(value) {
```
Comment (mstemm) [inline lib/jsdp-value-converter.js]: The term ToJuttle is a bit confusing to me. Isn't this really converting to juttle-compatible javascript types? Maybe we could use juttle compatible or something instead.

Comment (go-oleg) [inline lib/jsdp-value-converter.js]: Yea, I had somewhat of a rough time with the naming. Will rename to `xxxJSDPValue` and `xxxJuttleValue`.

**Comment by mstemm** [comment]
lgtm!

**Comment by go-oleg** [comment]
Going to merge this into the support-juttle-0.3.0 branch instead of master and to bump the juttle-client-version there to something that has jsdp support.

Closing.

**Pull request closed without merging**